platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/AdminAlertImpl.java

```diff
@@ -75,6 +76,15 @@ public AdminAlertImpl(PersistentStore persistentStore, EventAdmin eventAdmin)
       }
     } catch (PersistenceException pe) {
       LOGGER.debug("Error retrieving system alert.", pe);
+
+      return Collections.singletonList(
+          new Alert(
+                  "solr",
```

Comment:
❗️  How do you know it is a solr issue here? What you are receiving is a `PersistenceException`. The fact that the current underlying implementation of this API is using Solr is unknown to you here.

Reply:
You are correct. I hope we are not mentioning Solr by name in any of the messages. If so, we should replace references to "Solr" with "persistent storage".

Reply:
Agreed. That being said, the whole PR was stating to have a Solr connectivity alert and that is not what we have done here.

Reply:
Let's make sure we rename this to something else than `"solr"`